routes/api/children.js

```diff
@@ -0,0 +1,8 @@
+const childrenController = require('../../controllers/childrenController');
+
+module.exports = (app) =>{
+    app.post('/api/addChild',childrenController.createChild);
```

Comment:
Your routes here are camelCased, but front end routes arent? Keep them consistent. I'm sure your front-end routes should be camel cased.